lib/apt-install-php-ubuntu.sh

```diff
@@ -41,7 +41,7 @@ then
          php${version}-zip
 fi
 
-if [[ $release = 'focal' && `echo "$version < 8.0" | bc` == 1 ]]
+if [[ $release = 'focal' && `echo "$version < 8.1" | bc` == 1 ]]
```

Comment:
```suggestion
if [[ $release = 'focal' && `echo "$version < 8.0" | bc` == 1 ]]
```
The JSON and xmlrpc extensions exist only under PHP 8.0. This fix is probably unnecessary.
https://github.com/nanasess/setup-php/runs/5008646783?check_suite_focus=true#step:7:512